hooks/post_gen_project.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if echo "{{ cookiecutter.initial_git_commit }}" | grep -iq '^y'; then
+    echo -e "\e[1;33m==> Intializing a new git repository \e[0m"
+    echo -e "\e[1;32m==> git init \e[0m"
+    git init
+    echo -e "\e[1;33m==> Adding basic .gitignore for a TeX project \e[0m"
+    echo -e "#Latex\n*.aux\n*.glo\n*.idx\n*.log\n*.toc\n*.ist\n*.acn\n*.acr\n*.alg\n*.bbl\n*.blg\n*.dvi\n*.glg\n*.gls\n*.ilg\n*.ind\n*.lof\n*.lot\n*.maf\n*.mtc\n*.mtc1\n*.out\n*.synctex.gz" >> .gitignore
+    echo -e "\e[1;32m==> git status \e[0m"
+    git status
+    echo -e "\e[1;33m==> Staging all project files \e[0m"
+    echo -e "\e[1;32m==> git add -A \e[0m"
+    git add -A
+    echo -e "\e[1;32m==> git status \e[0m"
+    git status
+    echo -e "\e[1;33m==> Making the initial commit \e[0m"
+    echo -e "\e[1;32m==> git commit \e[0m"
+    git commit -m "Create base project"
+    echo -e "\e[1;32m==> git log \e[0m"
+    git log
+    echo -e "\e[1;32m==> git status \e[0m"
+    git status
+    if [ $(echo "{{ cookiecutter.enter_git_origin }}" | grep -i '[a-z]' | wc -c) -gt 4 ]; then
+        echo -e "\e[1;33m==> Setting up {{ cookiecutter.enter_git_origin }} as origin \e[0m"
```

Comment:
All this is breaking UI:

<img width="621" alt="Screenshot 2019-07-23 at 5 46 11 PM" src="https://user-images.githubusercontent.com/8039608/61711579-fed82a80-ad71-11e9-90fc-27f8c295b9be.png">